zerver/views/upload.py

```diff
@@ -14,6 +15,9 @@
 from zerver.models import UserProfile
 from django.conf import settings
 
+from zerver.lib.rest import rest_dispatch as _rest_dispatch
+rest_dispatch = csrf_exempt((lambda request, *args, **kwargs: _rest_dispatch(request, globals(), *args, **kwargs)))
+
```

Comment:
You no longer need to do this as of the recently merged refactoring of our urls.py files.